draft-kazuho-httpbis-priority.md

```diff
@@ -442,10 +442,11 @@ server and the intermediary.
 
 Once a client sends a request, circumstances might change and mean that it is
 beneficial to change the priority of the response. As an example, a web browser
-might issue a prefetch request for an HTML document with the urgency parameter
-of the Priority request header field set to `background`. Then, when the user
-navigates to the HTML while prefetch is in action, it would send a
-reprioritization frame with the priority field value set to `urgency=0`.
+might issue a prefetch request for a JavaScript file with the urgency parameter
+of the Priority request header field set to `urgency=6` (background). Then, when
+the user navigates to a page which references the new JavaScript file, while the
+prefetch is in progress, the browser would send a reprioritization frame with the
+priority field value set to `urgency=0` (default).
```

Comment:
Should the urgency be -1, assuming that this JavaScript file is render-blocking?

Reply:
I think both are fine, since JS is not necessarily render blocking, not even if it's prefetched. Yet, usually it is, and I feel it's best to encourage people to give JS high priorities, so updated in fad2b3c.